packages/block-editor/src/components/block-inspector/index.js

```diff
@@ -92,50 +136,31 @@ function BlockInspector() {
 	const blockInspectorAnimationSettings =
 		useBlockInspectorAnimationSettings( blockType );
 
-	const borderPanelLabel = useBorderPanelLabel( {
-		blockName: selectedBlockName,
-	} );
+	const hasSelectedBlocks = selectedBlockCount > 1;
 
-	const hasSelectedBlocks = count > 1;
-
-	if ( hasSelectedBlocks && ! isSectionBlockInSelection ) {
+	if (
+		hasSelectedBlocks &&
+		! isSectionBlockInSelection &&
+		! isSectionBlock
+	) {
 		return (
 			<div className="block-editor-block-inspector">
 				<MultiSelectionInspector />
-				{ showTabs ? (
+				{ hasMultipleTabs ? (
 					<InspectorControlsTabs tabs={ availableTabs } />
 				) : (
-					<>
-						<InspectorControls.Slot />
-						<InspectorControls.Slot
-							group="color"
-							label={ __( 'Color' ) }
-							className="color-block-support-panel__inner-wrapper"
-						/>
-						<InspectorControls.Slot
-							group="background"
-							label={ __( 'Background image' ) }
-						/>
-						<InspectorControls.Slot
-							group="typography"
-							label={ __( 'Typography' ) }
-						/>
-						<InspectorControls.Slot
-							group="dimensions"
-							label={ __( 'Dimensions' ) }
-						/>
-						<InspectorControls.Slot
-							group="border"
-							label={ borderPanelLabel }
-						/>
-						<InspectorControls.Slot group="styles" />
-					</>
+					<StyleInspectorSlots
+						blockName={ selectedBlockName }
+						showAdvancedControls={ false }
+						showPositionControls={ false }
+						showBindingsControls={ false }
+					/>
 				) }
 			</div>
 		);
 	}
 
-	if ( hasSelectedBlocks && isSectionBlockInSelection ) {
+	if ( hasSelectedBlocks && isSectionBlockInSelection && ! isSectionBlock ) {
```

Comment:
Here `isSectionBlockInSelection` seems to contradict `! isSectionBlock`. Which one should it be? 😅